src/screens/Market/index.tsx

```diff
@@ -335,61 +357,63 @@ export default function Market({ navigation }: { navigation: any }) {
 
   const renderEmptyComponent = useCallback(
     () =>
-        search ? ( // shows up in case of no search results
```

Comment:
This whole part is just lint

Reply:
I hope you didn't sneak in an easter egg hoping we just trust you and don't review it 👀